staff/robert-anthony/precurso/master/styles/styles.css

```diff
@@ -0,0 +1,229 @@
+/*
+font-family: 'Faster One', cursive;
+font-family: 'Codystar', cursive;
+font-family: 'Sarina', cursive;
+font-family: 'Monofett', cursive;
+font-family: 'Henny Penny', cursive;
+font-family: 'Nova Mono', monospace;
+font-family: 'Bad Script', cursive;
+font-family: 'Fira Mono', monospace;
+
+*/
+.textSnazz, #scoreBoard #cerrar {
```

Comment:
suggestion: for css selector names consider using chain-case instead of camelCase

Reply:
👍🏻